### ISSUE TITLE
Cannot pull from full lunar water cauldrons with glass bottles

### ISSUE THREAD

**Issue by tyon2006** [Ellpeck/Nyx#15]
Hey, I found that you cannot fill empty bottles with lunar water from a cauldron and you cannot fill a cauldron back up with lunar water from a lunar water bottle, on the latest release. It consumes the bottle and liquid in the cauldron, but does not produce a lunar water bottle.

Buckets seem to add and remove with no problem.

**Comment by tyon2006** [comment]
I have an additional finding. If you let the lunar water sit overnight in a cauldron again, the sparkles begin again. If you throw in a lapis, the sparkles stop, the block state on the cauldron does not change, but then you can remove the lunar water as normal in 3 bottles. 

The lunar water example from above was added to the cauldron from a lunar water bucket, not created naturally.

**Comment by Ellpeck** [comment]
Are you.. in creative when trying to drain using the bottle?

**Comment by tyon2006** [comment]
Hey, thanks for the update push. I tried this again with the new version and it will not push or pull lunar water to/from the cauldron into a glass bottle. I tried this in and out of creative mode. Buckets continue to work as expected. 

If its testing correctly on your end, I suspect this is an interaction with Inspirations. However, inspirations seems to work with other modded potions and non-potion liquids that can be bottled. 

Further research indicates that the bottles that are working are stylized as <minecraft:potion>.withtag{[potion effect name]}. I suspect that's why the bucket works and the bottle does not. 

As for the stacks, now it seems like the lunar water bottle will no longer stack beyond 1. It is not responding to the change that Quark applies, so I cannot test the stack size change.
